nilearn/glm/contrasts.py

```diff
@@ -262,6 +265,35 @@ def p_value(self, baseline=0.0):
         self.p_value_ = p_values
         return p_values
 
+    def one_minus_pvalue(self, baseline=0.0):
+        """Return a parametric estimate using
+        the cumulative distribution function of the p-value associated
```

Comment:
isn't it the cdf of the computed statistic, rather than of the p-value? or maybe say it returns 1 - the pvalue, but computed differently for numerical stability

Reply:
OK, I changed this docstring accordingly, too.